lib/travis/build/script.rb

```diff
@@ -78,6 +79,12 @@ def cache_slug
         'cache'
       end
 
+      def archive_url_for(bucket, version, ext = 'bz2')
+        lang = self.class.name.split('::').last.downcase
+
+        "https://s3.amazonaws.com/#{bucket}/binaries/#{host_os}/#{rel_version}/$(uname -m)/#{lang}-#{version}.tar.#{ext}"
```

Comment:
Is `uname -m` the one that's reliably going to be `x86_64` for 64-bit, or is there a chance it'll be `amd64`?